### PR TITLE
enh(Graphs): Move graph options to Graph header

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This moves to "Graph options" gear to graphs header
https://www.loom.com/share/4058c49eaee4485c91dfed2204eae5b8

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Pre-requisite: Have at least a service with graph datas
- Click on job row
- Select "Graph" tab on the panel
- -> The Graph options icon is displayed on the right side of the graphs header
- Click on the gear icon
- Enable "Disaply tooltips"
- Hover graph datas  with your mouse
- -> A tooltips with metrics value is displayed

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by Thebarda** [comment]
The popover is now anchroed to the top of the button : https://www.loom.com/share/5f18a908cb7a456a999ba8ff5d053ff9

**Comment by bdauria** [comment]
Nice 👍 

Can we maybe switch the colors from black to primary inside the "time" button (for both labels and icon) in order to stay coherent with the rest?

**Comment by Thebarda** [comment]
> Can we maybe switch the colors from black to primary inside the "time" button (for both labels and icon) in order to stay coherent with the rest?
> 
> I would add also that know that we regained the missing spacing, we can restore the labels to their original text (Last day instead of 1D etc...)

Here it is
newGraphHeader2

The original time periods labels doesn't fit with this layout

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -14367,16 +14367,7 @@ msgstr "Permitir certificado autofirmado"
...
-msgstr ""
-
-msgid "31 D"
-msgstr ""
-
-msgid "24 hours"
+msgid "1 day"
 msgstr ""
 
 msgid "7 days"
```
Comment (bdauria) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]: Uncommented msgid / msgstr will be translated as empty strings

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16100,14 +16091,17 @@ msgid "The end date must be greater than the start date"
 msgstr "La date de fin doit être supérieure à la date de début"
 
 msgid "Graph options"
-msgstr "Options de graphe"
+msgstr "Options de graph"
```
Comment (Thebarda) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16094]:
```suggestion
msgstr "Options de graphe"
```

**Review by bdauria — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**